conf/geronimo.yml

```diff
@@ -58,8 +58,8 @@ issue_tracker:
     domain: https://issues.apache.org/jira
     project_key: GERONIMO
     # Download using `download_jira_data.Rmd`
-    issues: ../../rawdata/issue_tracker/geronimo_issues.json
-    issue_comments: ../../rawdata/issue_tracker/geronimo_issue_comments.json
+    issues: ../rawdata/issue_tracker/geronimo_issues.json
+    issue_comments: ../rawdata/issue_tracker/geronimo_issue_comments.json
```

Comment:
This is another file that should not be modified. I also recommend you organize the folders as kaiaulu/kaiaulu so you do not need to keep switching the `../` back and forth every time.